backend/core/jobs/sample.py

```diff
@@ -0,0 +1,16 @@
+from django_extensions.management.jobs import BaseJob
+
+from core.user.models import User
+
+
+class Job(BaseJob):
+    """
+    Sample for jobs.
```

Comment:
I guess this file can be removed?

Reply:
This is a sample job for creating new jobs. For beginners it is a good starting point so i would leave it here.